### PR TITLE
Colors and icons

### PR DESCRIPTION
**Pull request by aliaksandr-savitski**
Please, do not merge it yet, as dark theme colors will follow. I wanted you to take a look at what is there already, and maybe leave a few comments ;)

Most of the changes concern icons update, so you can skip them ;)

1. The most important stuff is `defaultTheme.js`, and probably will cause the biggest discussion. I've added gray colors to `colors` constant, and my suggestion not to use it inside object that we export. I suggest using `theme` variable which is created. The idea is to define more or less meaningful names to those color and use them inside the project so we don't worry if any `color` property changes. If it's not really clear - don't hesitate to contact me.

I myself found it making a development process less nice, but I can live with that - will be interesting to hear your opinion. 

2. Changed `icons.js` a little bit so we have a name of the icon component in storybook. I found it hard to find appropriate icon if it's somewhere in the middle.

### CODE REVIEW

File: src/themes/helpers.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  let green = (num & 0x0000FF) + value;
+  if (green > 255) green = 255;
+  else if (green < 0) green = 0;
+
+  // Blue
+  let blue = ((num >> 8) & 0x00FF) + value;
+  if (blue > 255) blue = 255;
+  else if (blue < 0) blue = 0;
+
+  return (usePound?"#":"") + (green | (blue << 8) | (red << 16)).toString(16);
```
Comment (rafalswietek) [inline src/themes/helpers.js]: Please, use apostrophes instead of quotation marks and add some spaces ;)

File: src/themes/DarkTheme.js [hunk 2]
```diff
@@ -1,16 +1,40 @@
+import { lightenDarkenColor } from './helpers';
+import { convertToRgb } from './helpers';
+
 const colors = {
```
Comment (kasperslu) [inline src/themes/DarkTheme.js]: Consider changing colors names according to http://chir.ag/projects/name-that-color

File: src/themes/DarkTheme.js [hunk 3]
```diff
@@ -1,16 +1,40 @@
...
+  darkerDark: '#2D2E33',
+  lighterDark: '#464651',
+  grey: '#CBCEDE',
+  lightGrey: '#E9EAED',
+  darkerGrey: '#878891',
   transparentWhite: 'rgba(255, 255, 255, 0.1)',
   transparentBlack: 'rgba(0, 0, 0, 0.1)',
 };
 
+const theme = {
```
Comment (kasperslu) [inline src/themes/DarkTheme.js]: Consider removing this layer

File: src/themes/DefaultTheme.js [hunk 4]
```diff
@@ -1,17 +1,43 @@
+import { lightenDarkenColor, convertToRgb } from './helpers';
+
 const colors = {
```
Comment (kasperslu) [inline src/themes/DefaultTheme.js]: Consider changing colors names according to http://chir.ag/projects/name-that-color

File: src/themes/DefaultTheme.js [hunk 5]
```diff
@@ -1,17 +1,43 @@
...
-  grey: '#CCD3DE',
-  darkGrey: '#3E4247',
+  grey: '#A1AAB7',
+  greyDark: '#5E646E',
+  greyDarker: '#44484F',
   transparentWhite: 'rgba(255, 255, 255, 0.1)',
   transparentBlack: 'rgba(0, 0, 0, 0.1)',
 };
 
+const theme = {
```
Comment (kasperslu) [inline src/themes/DefaultTheme.js]: Consider removing this layer

File: stories/Icons.js [hunk 6]
```diff
@@ -94,81 +73,6 @@ storiesOf('Icons', module)
...
-      <ViewGridIcon />
-      <ViewListIcon />
-      <WalletIcon />
-      <WarningIcon />
-      <WatchFolderIcon />
-      <ZoomCloseIcon />
-      <ZoomInIcon />
-      <ZoomOutIcon />
-      <ZoomResetIcon />
+      {iconsArray.map(getIcon)}
```
Comment (rafalswietek) [inline stories/Icons.js]: Can we run here a function called `getIcons()` that will run this `iconsArray.map()` for us, and also do the job that `getIcon` is doing right now? ;)

File: stories/Icons.js [hunk 7]
```diff
@@ -1,90 +1,69 @@
...
-  ZoomOutIcon,
-  ZoomResetIcon,
-} from '../src/components/icons';
+import defaultTheme from '../src/themes/DefaultTheme';
+import * as icons from '../src/components/icons';
+
+const iconsArray = Object.keys(icons).map((key) => icons[key]);
+
+function getIcon(Icon) {
+  var _this = <Icon />;
```
Comment (rafalswietek) [inline stories/Icons.js]: We don't use `var`s anymore and we don't use variable names starting with the underscore.
Also I would change that `this` into something like `iconComponent` ;)

File: src/themes/DarkTheme.js [hunk 8]
```diff
@@ -1,14 +1,18 @@
+import { lightenDarkenColor } from './helpers';
+import { convertToRgb } from './helpers';
```
Comment (rafalswietek) [inline src/themes/DarkTheme.js]: Could we make a single import from those two? :P

File: src/themes/helpers.js [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+  let green = (num & 0x0000FF) + value;
+  if (green > 255) green = 255;
+  else if (green < 0) green = 0;
+
+  // Blue
+  let blue = ((num >> 8) & 0x00FF) + value;
+  if (blue > 255) blue = 255;
+  else if (blue < 0) blue = 0;
+
+  return (usePound?'#':'') + (green | (blue << 8) | (red << 16)).toString(16);
```
Comment (rafalswietek) [inline src/themes/helpers.js]: Please, add spaces here
```suggestion
  return (usePound ? '#' : '') + (green | (blue << 8) | (red << 16)).toString(16);
```

File: src/themes/DefaultTheme.js [hunk 10]
```diff
@@ -20,7 +25,13 @@ const gradients = {
   brightActive: 'linear-gradient(0deg, #FFF 0%, #F8F9FC 100%)',
 };
 
+const shadows = {
+  shadowColor: `rgba(${convertToRgb(colors.grayChateau)}, 0.3)`,
+}
+
 export default {
+  convertToRgb,
```
Comment (rafalswietek) [inline src/themes/DefaultTheme.js]: Why would we want to export that `convertToRgb` from here?

**Pull request merged**